sdk/rms_sdk/UnitTests/profile_ut/profile_ut.pro

```diff
@@ -16,9 +16,9 @@ LIBS += -L$$REPO_ROOT/bin
 
 CONFIG(debug, debug|release) {
     TARGET = $$join(TARGET,,,d)
-    LIBS += -lrmsd
+    LIBS += -lrmsd -lrmscryptod
```

Comment:
Does profile_ut depend on rmscrypto?

Reply:
On Windows Qt was able to find the dependencies but on Linux I got the error that librms.so could not find librmscrypto.so